main.tf

```diff
@@ -20,6 +20,8 @@ module "lambda" {
   environment = {
     variables = {
       SLACK_WEBHOOK = var.hook_url
+      LOG_LEVEL     = var.LOG_LEVEL
```

Comment:
The `LOG_LEVEL` terraform variable is lowercase in its variable definition.

Reply:
fixed